TwigExtension.php

```diff
@@ -228,4 +231,9 @@ public function getName()
     {
         return 'nelmio_js_logger';
     }
+
+    private function generateUrl()
+    {
+        return addslashes($this->router->generate('nelmio_js_logger_log', [], $this->useAbsoluteUrl ? UrlGeneratorInterface::ABSOLUTE_URL : UrlGeneratorInterface::ABSOLUTE_PATH));
```

Comment:
`addslashes` should not be part of a method named `generateUrl`, as it is not about generating the URL but about escaping it for usage in JS (and so should stay in the function using the URL in a JS context)

Reply:
returned `addslashes` back to `initErrorLogger` and `initLogger`